### ISSUE TITLE
Stop using skimdb

### ISSUE THREAD

**Issue by zeke** [zeke/all-the-packages#2]
http://blog.npmjs.org/post/157615772423/deprecating-the-all-registry-endpoint

> In six months, on September 1 2017, we plan to shut down the servers that back the GET /-/all endpoint of registry.npmjs.org. 

This package will need to be updated to use a different technique for collecting package data at install time, probably using [package-stream](https://github.com/nice-registry/package-stream).

Thanks to @rmg for pointing this out in https://github.com/npm/registry/issues/136

**Comment by rootAvish** [comment]
Hey @zeke, does this mean the data that this package downloads has not been updated after September?

**Comment by zeke** [comment]
@rootAvish that's correct. You may want to take a look at https://github.com/nice-registry/package-stream to easily download all the package data.

**Comment by bananabr** [comment]
Hi @zeke,

I don't think you guys ever replaced skimdb right?
Can one still use this package somehow?

Thanks

**Comment by zeke** [comment]
Yeah this package is unmaintained. It would need to be updated to use the `replicate.npmjs.com` endpoint.
